internal/inventory/asset.go

```diff
@@ -37,9 +37,10 @@ type AssetSubCategory string
 const (
 	SubCategoryAuthorization AssetSubCategory = "authorization"
 	SubCategoryCompute       AssetSubCategory = "compute"
-	SubCategoryStorage       AssetSubCategory = "storage"
 	SubCategoryDatabase      AssetSubCategory = "database"
+	SubCategoryIntegration   AssetSubCategory = "integration"
```

Comment:
I'm not completely sure about this sub-category. I personally would call it `messaging`. But maybe just a quick check with other engineers might solve our question :D

Reply:
Yeah, I agree. I'm up for a discussion. Any recommendations welcome.

Reply:
@tinnytintin10, any preference towards the classification of SNS Topics/Subscriptions?

Reply:
I vote for `messaging`. How I'd decide it: have a read on how analog techs (pubsub, kafka, rabbitmq and others) are called. If there is a common name, let's use what the market is using